docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -201,6 +201,41 @@ to the filter:
     ALTER TABLE test_partitioned_table EXECUTE optimize
     WHERE partition_key = 1
 
+expire_snapshots
+~~~~~~~~~~~~~~~~
+
+The ``expire_snapshots`` command removes all snapshots and all related metadata and data files.
+Regularly expiring snapshots is recommended to delete data files that are no longer needed,
+and to keep the size of table metadata small.
+The age of snapshots that will be expired is defined by ``retention_threshold`` parameter.
```

Comment:
The procedure affects all snapshots that are older than the time period configured with the ``retention_threshold`` parameter.